lib/Sema/TypeCheckPropertyWrapper.cpp

```diff
@@ -481,6 +480,9 @@ AttachedPropertyWrappersRequest::evaluate(Evaluator &evaluator,
       continue;
     }
 
+    // Note: Getting the semantic attrs here would trigger a request cycle.
+    auto attachedAttrs = var->getAttrs();
```

Comment:
Since the set of attributes returned by getAttrs() now depends on other request evaluation having taken place, maybe we could also add a getParsedAttrs() to only return parsed attributes? Then over time, existing calls to getAttrs() can migrate over to use getCustomAttrs() or getParsedAttrs().

Reply:
Yeah, I think that makes sense. I believe the `getOriginalAttrs()` accessor that Ben added in https://github.com/apple/swift/pull/63738 is equivalent to the `getParsedAttrs()` you're describing.